### PR TITLE
NI-6450: add gcp support s3s2

### PR DESCRIPTION
**Pull request by iostreamdoth**

### CODE REVIEW

File: aws_helpers/s3helper.go [hunk 1]
```diff
@@ -9,114 +9,128 @@ import (
 	session "github.com/aws/aws-sdk-go/aws/session"
 	options "github.com/tempuslabs/s3s2/options"
 	utils "github.com/tempuslabs/s3s2/utils"
+    gcp_helpers "github.com/tempuslabs/s3s2/gcp_helpers"
```
Comment (rsingh2507) [inline aws_helpers/s3helper.go]: indentation

File: python_example.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        GoString(pubKey.encode('utf-8'), len(pubKey)),
+        GoString(privKey.encode('utf-8'), len(privKey)),
+        GoString(ssmPubKey.encode('utf-8'), len(ssmPubKey)),
+        GoString(ssmPrivKey.encode('utf-8'), len(ssmPrivKey)),
+        True,
+        10
+    )
+except Exception as ex:
+    raise ex
+print("done execution")
```
Comment (rsingh2507) [inline python_example.py:56]: print?

Comment (iostreamdoth) [reply python_example.py:56]: this is python example script for so object

File: aws_helpers/s3helper.go (hunk 1 above)
Comment (iostreamdoth) [reply aws_helpers/s3helper.go]: fixed

Comment (iostreamdoth) [reply aws_helpers/s3helper.go]: this one was odd...

**Review by rsingh2507 — APPROVED** [review]

**Pull request merged**